<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmlDesigner::AnnotationEditorDialog</class>
 <widget class="QDialog" name="QmlDesigner::AnnotationEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="statusContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="statusAddButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="statusComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>In Progress</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>In Review</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Done</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbTabView">
        <property name="text">
         <string>Tab View</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbTableView">
        <property name="text">
         <string>Table View</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="annotationContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="targetIdLayout">
        <item>
         <widget class="QLabel" name="targetIdLabel">
          <property name="text">
           <string>Selected component</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="targetIdEdit">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="customIdLayout">
        <item>
         <widget class="QLabel" name="customIdLabel">
          <property name="text">
           <string>Custom ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="customIdEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="AnnotationTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="AnnotationTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnnotationTabWidget</class>
   <extends>QTabWidget</extends>
   <header>annotationtabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnnotationTableView</class>
   <extends>QTableView</extends>
   <header>annotationtableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>targetIdEdit</tabstop>
  <tabstop>customIdEdit</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QmlDesigner::AnnotationEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QmlDesigner::AnnotationEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
